README.md

```diff
@@ -1,3 +1,24 @@
-# Laundry Time
+# README
 
-August 2017 Craft Academy Mid Course Project
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```

Comment:
I would make sure that the README is restored to include info about your project